apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/eobByPatientIdAll.json

```diff
@@ -3950,9 +3950,8 @@
         },
         "modifier" : [ {
           "coding" : [ {
-            "system" : "https://bluebutton.cms.gov/resources/variables/rev_cntr_ndc_qty_qlfr_cd",
-            "code" : "ML",
-            "display" : "Milliliter"
```

Comment:
Dropping the same feedback here for posterity. As discussed, changes where we're removing fields probably qualify as a backward incompatible, and we'll need to maintain this idiosyncratic behavior rather than removing it and breaking expectations.

Reply:
Resolved!